cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/application/lifecycle/ApplicationLifecycleController.java

```diff
@@ -37,7 +37,7 @@
      * time must cover the completion of processing current REST requests. This time must also cover the maximum queue
      * poll time for message listeners and completion of processing for all messages already received.
      */
-    static final long SHUTDOWN_TIMEOUT_MILLIS = 50000; // = 50s
+    static final long SHUTDOWN_TIMEOUT_MILLIS = 60000; // = 60s
```

Comment:
I suggested 65 seconds, not 60. This is because the poll time for Step Function activities are 60 seconds. Above the 60 seconds, we also need to leave time for the processing of any activity returned.